Makefile

```
@@ -56,3 +56,8 @@ run: check
 .PHONY: error
 error:
 	go run github.com/layer5io/meshkit/cmd/errorutil -d . analyze -i ./helpers -o ./helpers
+
+ # runs a local instance of nats server in detached mode
+PHONY: run-nats
+run-nats:
```

```suggestion
PHONY: nats
nats:
```

We have a target naming convention of noun-verb (e.g. `nats-run`). In the even t of __run__, though, we simply drop it and assume __run__ is the default behavior.